Sources/SwiftGodot/MacroDefs.swift

```diff
@@ -5,7 +5,7 @@
 //  Created by Miguel de Icaza on 9/25/23.
 //
 
-#if !os(Windows)
+#if !(os(Windows) && swift(>=5.9))
```

Comment:
This will enable macros for Windows with Swift < 5.9

Reply:
good catch I got myself mixed up